src/main/java/com/amazon/dlic/auth/http/jwt/keybyoidc/KeySetRetriever.java

```diff
@@ -65,6 +66,16 @@ public class KeySetRetriever implements KeySetProvider {
 		}
 	}
 
+    KeySetRetriever(SSLConfig sslConfig, boolean useCacheForOidConnectEndpoint, String jwksUri) {
+        this.sslConfig = sslConfig;
+        this.jwksUri = jwksUri;
+
+        if (useCacheForOidConnectEndpoint) {
```

Comment:
Is it possible to separate this out here and above to avoid duplicating code?

Reply:
Sure, I will fix this